showcase/src/app/demo-ui/components/table-with-items-per-page-selector/table-with-items-per-page-selector.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, Inject } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { StarkPaginationConfig, StarkTableColumnProperties, StarkTableFilter, StarkTableRowActions } from "@nationalbankbelgium/stark-ui";
+
+const DUMMY_DATA: object[] = [
+	{
+		id: 1,
+		title: { label: "first title (value: 1)", value: 1 },
+		description: "first one",
+		extra: "number one has some extra information"
```

Comment:
Is the `extra` property useful here ? Or is it linked to the implementation of collapsable rows in the table ?
If so, could you please remove it since it's not related to the items-per-page ?

Reply:
No, Actually it's coming from the table-regular example.

Reply:
I looked at the code of regular table and it is there to show the feature "show/hide column".
We should keep it for the regular table 😊

![](https://i.imgur.com/OcyzIAC.png)

Reply:
so let's keep it :-)